src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  background: #fff;
+  font-size: 20px;
+  flex-direction: column;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.2rem;
+}
+
+.completed {
+  text-decoration: line-through;
+  color: #f00;
+  font-weight: bold;
+}
+
+input[type='checkbox'] {
+  width: 1.4rem;
+  height: 1.2rem;
+}
+
+#task-form input {
+  font-size: 1.2rem;
+  padding: 0.5rem;
+  border-radius: 5px;
+}
+
+#add-task-btn {
+  background: #000;
+  color: #fff;
+  border: none;
+  font-size: 1.2rem;
+  padding: 0.5rem 1.4rem;
+}
+
+ul li {
+  display: flex;
+  align-items: center;
+
+  /* justify-content: space-between; */
+  list-style: none;
+  font-size: 1.5rem;
+  margin: 0.9rem;
+
+  /* width: 100%; */
```

Comment:
- [Optional] Kindly remove all commented pieces of code from your documents to keep your documents clean.